src/scss/obsidian/file-nav.scss

```diff
@@ -64,6 +64,7 @@ body:not(.is-mobile) .nav-folder.mod-root>.nav-folder-title .nav-folder-title-co
 .nav-file-title.is-active,
 .nav-folder-title.is-active,
 .nav-file-title.is-being-dragged,
+body:not(.is-grabbing) .nav-folder-title.is-active:hover,
 body:not(.is-grabbing) .nav-folder-title:hover,
```

Comment:
Not sure if the original selector can be replaced by `.is-active` version, keeping it here for now